### ISSUE TITLE
EMI Loot fails to open Vailla direct drops loot tables

### ISSUE THREAD

**Issue by afzalarsalan** [fzzyhmstrs/EMI_loot#55]
When running EMI Loot with Minecraft 1.20.1 and Neoforge 47.1.101, EMI loot has an error figuring out the loot tables. 

```
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/creeper.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/drowned.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/evoker.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/husk.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/piglin.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/piglin_brute.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/pillager.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/skeleton.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/stray.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/vindicator.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/wither.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/wither_skeleton.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/zombie.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/zombie_villager.json
[21:34:37] [main/ERROR] [emi_loot/]: Failed to open or read direct drops loot table file: minecraft:direct_drops/entities/zombi
```

**Comment by unilock** [comment]
Should be fixed in v0.6.6+fix1. https://modrinth.com/mod/emi-loot/version/0.6.6+fix1+1.20.1+forge

**Comment by fzzyhmstrs** [comment]
Closing as fixed
